app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] Since you are using `author_id_id` as foreign_key you need to make sure to use ` belongs_to :author_id` for the model's associations as you did in the others models. Kindly correct this ✍🏻 
### - Note: since it's the only thing you're missing, I approve of this PR but I highly recommend you fix it 💯 👍🏻

Reply:
Thank you very much for your review.